hera_mc/geo_handling.py

```diff
@@ -302,7 +286,7 @@ def get_location(self, to_find, query_date, station_types=None):
     def print_loc_info(self, loc, verbosity='h'):
```

Comment:
A brief docstring indicating the meaning of the return value would be helpful, even if this function isn't part of the M&C/CM API.